### PR TITLE
Added a button to link docs to appPage

### LINKED ISSUE #313: [FEATURE] Button to go back to app from Docs page
### Description

There is no button to close the docs page once it is opened. Clicking on the icon redirects us back to the homepage and not to the app page.

### Screenshots

image

### Additional information

Suggestion: 
Add a cross button to close the docs page

### PR DESCRIPTION
**Pull request by harshit-002**
## Fixes Issue

Closes #313 

## Changes proposed
Add a button that on click redirects to appPage in the docs section of the website.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
image

### CODE REVIEW

File: src/routes/docs/+layout.svelte [hunk 1]
```diff
@@ -12,6 +12,12 @@
   </div>
   <div class="w-full border-gray-300 dark:border-gray-600 lg:w-3/5 lg:border-x lg:px-8">
     <slot />
+    <div class="flex" style=" justify-content: space-around; margin-top:5%;">
+      <a
+        class="default-transition text-skin-text-highlight hover:underline hover:decoration-inherit"
```
Comment (Cahllagerfeld) [inline src/routes/docs/+layout.svelte:17]: I think we should add the link to `/app` to the topbar, what do you think?

**Review by Cahllagerfeld — COMMENTED** [review]
A fix for the failing test is in main. So when merging main into this, the tests should pass again

File: src/routes/docs/+layout.svelte (hunk 1 above)
Comment (harshit-002) [reply src/routes/docs/+layout.svelte:17]: yeah, then i think i should name it just "AppPage" and place it in topbar, or any other name you suggest.
image

**Comment by harshit-002** [comment]
> A fix for the failing test is in main. So when merging main into this, the tests should pass again

sorry, can you explain a bit more why the test failed, i'm a newbie.

**Comment by Cahllagerfeld** [comment]
To be honest, I'm not 100% sure because the tests weren't changed. I think it relates to a self-closing popup, but its just a guess

File: src/lib/components/header.svelte [hunk 2]
```diff
@@ -21,7 +21,10 @@
 
   export let username: string | undefined;
 
-  const navItems = [{ name: 'Docs', path: '/docs' }];
+  const navItems = [
+    { name: 'Docs', path: '/docs' },
+    { name: 'AppPage', path: '/app' },
```
Comment (Cahllagerfeld) [inline src/lib/components/header.svelte:26]: Shall we just call it `App` or something? I'm not sure about this camle case here?

**Review by Cahllagerfeld — APPROVED** [review]
Looks good to me 👍

**Pull request merged**